src/Idris/Coverage.hs

```diff
@@ -457,15 +457,10 @@ checkTotality path fc n
                          putIState i { idris_totcheckfail = (fc, msg) : idris_totcheckfail i}
                          addIBC (IBCTotCheckErr fc msg)
 
-    warnPartial n t
-       = do i <- getIState
-            case lookupDef n (tt_ctxt i) of
-               [x] -> do
-                  iWarn fc . pprintErr i . Msg $ "Warning - " ++ show n ++ " is " ++ show t
---                                ++ "\n" ++ show x
---                   let cg = lookupCtxtName Nothing n (idris_callgraph i)
---                   iputStrLn (show cg)
-
+    warnPartial n t = do
```

Comment:
This isn't removing a redundant "do", it's just putting it in dangling position, which isn't the style that most of Idris is written in.